docs/DeveloperGuide.adoc

```diff
@@ -729,11 +729,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |search for the user based on certain keywords|find contacts easily
```

Comment:
this is not a high priority user story. they can already search by name; adding one more way is nice to have but not 'must have'. High priority stories are those that are necessary to make a minimum viable product. Review priorities of all stories